### PR TITLE
PMM-12460 PMM dump

### PR DESCRIPTION
**Pull request by idoqo**
https://jira.percona.com/browse/PMM-12460

- [ ] https://github.com/percona/pmm/pull/2493
- [x] https://github.com/percona/pmm-dump/pull/66
- [x] https://github.com/percona/pmm-dump/pull/82
- [ ] https://github.com/percona-platform/grafana/pull/694

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-33d06f1
client docker - perconalab/pmm-client-fb:PR-3407-33d06f1
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-33d06f1.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-33d06f1&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-33d06f1.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-9063a58
client docker - perconalab/pmm-client-fb:PR-3407-9063a58
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-9063a58.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-9063a58&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-9063a58.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14294/

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14294/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-25c0ac1
client docker - perconalab/pmm-client-fb:PR-3407-25c0ac1
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-25c0ac1.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-25c0ac1&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-25c0ac1.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14312/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-b84743c
client docker - perconalab/pmm-client-fb:PR-3407-b84743c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-b84743c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-b84743c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-b84743c.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14324/

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-f4c6f66
client docker - perconalab/pmm-client-fb:PR-3407-f4c6f66
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-f4c6f66.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-f4c6f66&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-f4c6f66.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-ad96782
client docker - perconalab/pmm-client-fb:PR-3407-ad96782
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-ad96782.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-ad96782&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-ad96782.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-0e34bbb
client docker - perconalab/pmm-client-fb:PR-3407-0e34bbb
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-0e34bbb.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-0e34bbb&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-0e34bbb.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-f25dad6
client docker - perconalab/pmm-client-fb:PR-3407-f25dad6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-f25dad6.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-f25dad6&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-f25dad6.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14338/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-31f05d3
client docker - perconalab/pmm-client-fb:PR-3407-31f05d3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-31f05d3.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-31f05d3&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-31f05d3.tar.gz

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-d534ba2
client docker - perconalab/pmm-client-fb:PR-3407-d534ba2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-d534ba2.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-d534ba2&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-d534ba2.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14401/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-96522ad
client docker - perconalab/pmm-client-fb:PR-3407-96522ad
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-96522ad.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-96522ad&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-96522ad.tar.gz

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-6405350
client docker - perconalab/pmm-client-fb:PR-3407-6405350
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-6405350.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-6405350&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-6405350.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-b991715
client docker - perconalab/pmm-client-fb:PR-3407-b991715
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-b991715.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-b991715&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-b991715.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14426/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-101e43c
client docker - perconalab/pmm-client-fb:PR-3407-101e43c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-101e43c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-101e43c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-101e43c.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-2515c1e
client docker - perconalab/pmm-client-fb:PR-3407-2515c1e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-2515c1e.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-2515c1e&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-2515c1e.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14438/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-1d2efc6
client docker - perconalab/pmm-client-fb:PR-3407-1d2efc6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-1d2efc6.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-1d2efc6&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-1d2efc6.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-a157e7b
client docker - perconalab/pmm-client-fb:PR-3407-a157e7b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-a157e7b.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-a157e7b&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-a157e7b.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-6c8752f
client docker - perconalab/pmm-client-fb:PR-3407-6c8752f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-6c8752f.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-6c8752f&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-6c8752f.tar.gz

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm-dump/pull/66

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm-dump/pull/66

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-cd75b3f
client docker - perconalab/pmm-client-fb:PR-3407-cd75b3f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-cd75b3f.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-cd75b3f&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-cd75b3f.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-cd75b3f
client docker - perconalab/pmm-client-fb:PR-3407-cd75b3f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-cd75b3f.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-cd75b3f&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-cd75b3f.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-de098a0
client docker - perconalab/pmm-client-fb:PR-3407-de098a0
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-de098a0.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-de098a0&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-de098a0.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-3752db8
client docker - perconalab/pmm-client-fb:PR-3407-3752db8
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-3752db8.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-3752db8&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-3752db8.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-14332c3
client docker - perconalab/pmm-client-fb:PR-3407-14332c3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-14332c3.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-14332c3&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-14332c3.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-cedcb99
client docker - perconalab/pmm-client-fb:PR-3407-cedcb99
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-cedcb99.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-cedcb99&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-cedcb99.tar.gz

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-4e83773
client docker - perconalab/pmm-client-fb:PR-3407-4e83773
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-4e83773.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-4e83773&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-4e83773.tar.gz

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm/pull/2493

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/grafana/pull/694

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/grafana/pull/694

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-28c0f7a
client docker - perconalab/pmm-client-fb:PR-3407-28c0f7a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-28c0f7a.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-28c0f7a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-28c0f7a.tar.gz

File: ci.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+deps:
+  - name: pmm
+    branch: PMM-12460-pmm-dump
+
+  - name: grafana
+    branch: PMM-12459
+
+  - name: pmm-dump
+    branch: main
```
Comment (artemgavrilov) [inline ci.yml]:
```suggestion
    branch: fix-multi-instance-dumps
```

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-574be02
client docker - perconalab/pmm-client-fb:PR-3407-574be02
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-574be02.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-574be02&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-574be02.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-60aedbf
client docker - perconalab/pmm-client-fb:PR-3407-60aedbf
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-60aedbf.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-60aedbf&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-60aedbf.tar.gz

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
PMM3 Submodules skipped

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-009fd36
client docker - perconalab/pmm-client-fb:PR-3407-009fd36
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-009fd36.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-009fd36&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-009fd36.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-009fd36
client docker - perconalab/pmm-client-fb:PR-3407-009fd36
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-009fd36.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-009fd36&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-009fd36.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3407-6be35b0
client docker - perconalab/pmm-client-fb:PR-3407-6be35b0
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-6be35b0.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3407-6be35b0&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3407-6be35b0.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14578/

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14579/

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14579/

**Pull request closed without merging**